### ISSUE TITLE
Finish causal relation presence fasttext experiments

### ISSUE THREAD

**Issue by svenvanderburg** [SSIML2021/SSIML2021#23]
Finish the causal relation presence experiments that use fasttext. See [this notebook](https://github.com/jfasnellens/SSIML2021/blob/cc0e77c6f4db54c5f9dfe417c7b9ed941672021b/notebooks/1.1-causal-relation-precense-fasttext.ipynb).

- [ ] Use fixed Kfold splits across all experiments. (use random seed, see #16 )
- [ ] I suggest to use 5-fold instead of 10-fold to speed up the experiments. I think it wouldn't hurt performance that much.
- [ ] Finish the experiments using pretrained embeddings.
- [ ] IMO you can get rid of all the classification reports and confusion matrices for the training data, it doesn't add much information.
